pkg/kube/inject/webhook.go

```diff
@@ -835,9 +835,40 @@ func (wh *Webhook) serveInject(w http.ResponseWriter, r *http.Request) {
 // parseInjectEnvs parse new envs from inject url path
 // follow format: /inject/k1/v1/k2/v2, any kv order works
 // eg. "/inject/cluster/cluster1", "/inject/net/network1/cluster/cluster1"
+// since character slash is used as separator to specify keys and values,
+// if a value needs to contain slashes such as an url, then the path can be
```

Comment:
I would remove the documentation about the old / format, and just document the new way. The / code is still there for backward compatibility but we don't need to document it, since it has limitations and therefore shouldn't be needed or used anymore.

Reply:
hmmm. I think the old way actually is easier to use in cases that one does not need to use slash in variable values. I suspect that is the majority of the cases, I think both ways should be kept.

Reply:
Neither format is hard to use. Explaining that there are two different formats and how they inter-operate, in itself, is more complicated than the new required format. @howardjohn WDYT?